### PR TITLE
Set arg make ObsidianQuickSwitch search order

### PR DESCRIPTION
**Pull request by bigzhu**
Use rg sort
* #149

### CODE REVIEW

**Comment by bigzhu** [comment]
* #177

@epwalsh The conflicts in the CHANGELOG have been resolved

**Review by epwalsh — CHANGES_REQUESTED** [review]
Hey @bigzhu, sorry for the slow response.. I was out with COVID for a week.

Having two options called `sort_by` and `order_by` seems pretty confusing to me. I recommend we just have a `sort_by` option and then a `sort_reversed` (bool) option.

**Comment by bigzhu** [comment]
Oh, @epwalsh  I didn't notice your reply.
I submitted the request again.
Ps: my NeoVim auto formated README.md.

File: lua/obsidian/command.lua [hunk 1]
```diff
@@ -385,7 +405,8 @@ command.quick_switch = function(client, data)
         util.implementation_unavailable()
       end
       -- Search with telescope.nvim
-      telescope.find_files { cwd = dir, search_file = "*.md" }
+      local opts = attach_find_command(client, { cwd = dir, search_file = "*.md" })
```
Comment (epwalsh) [inline lua/obsidian/command.lua:408]: We should support `fzf.vim` and `fzf-lua` too.

File: lua/obsidian/command.lua [hunk 2]
```diff
@@ -5,6 +5,25 @@ local echo = require "obsidian.echo"
...
+  local sort_by = client.opts.sort_by
+  if sort_by == nil then
+    return opts
+  end
+  local sort_reversed = client.opts.sort_reversed
+  local sort = "sortr" -- default sort is reverse
+  if sort_reversed == false then
+    sort = "sort"
+  end
+  opts.find_command = { "rg", "--files", "--" .. sort, sort_by }
```
Comment (epwalsh) [inline lua/obsidian/command.lua:24]: We should start with `util.FIND_CMD` command and the `--sort/--sortr` options to that.

**Review by epwalsh — COMMENTED** [review]
Hey @bigzhu, thanks for those changes. I just have a couple more requests.

File: lua/obsidian/command.lua (hunk 2 above)
Comment (bigzhu) [reply lua/obsidian/command.lua:24]: `find_command` is the `telescope.find_files` parameter, I don't understand it has any business with `util.FIND_CMD`

**Comment by epwalsh** [comment]
I tried to push a commit to your branch and GitHub automatically closed this PR. Not sure what's going on.

**Comment by epwalsh** [comment]
Had to create a new PR here: #200

**Comment by bigzhu** [comment]
no problem

**Pull request closed without merging**